src/app/components/LazyImage/index.js

```diff
@@ -20,6 +24,35 @@ function LazyImage({ src, alt, placeholder, className, imageClassName, ...rest }
             onError={() => setIsLoaded(false)}
             {...rest}
           />
+          <div className={cx(styles.container_child)}>
+            <div className={cx(styles.container_child_logo)}>
+              <img src={IconBanner} alt="icon-logobanner"></img>
+              <div className={cx(styles.container_child_cnt)}>
+                <p>
+                  <b>5 năm</b>
+                </p>
+                <p>Tổ chức các kỳ thi</p>
+              </div>
+            </div>
+            <div className={cx(styles.container_child_users)}>
+              <img src={IconUsers} alt="icon-users"></img>
+              <div className={cx(styles.container_child_cnt)}>
+                <p>
+                  <b>3800+</b>
+                </p>
+                <p>Thành viên</p>
+              </div>
+            </div>
+            <div className={cx(styles.container_child_medal)}>
+              <img src={IconMedal} alt="icon-medal"></img>
+              <div className={cx(styles.container_child_cnt)}>
+                <p>
+                  <b>Nền tảng lập trình thi đấu</b>
+                </p>
+                <p>hàng đầu Việt Nam</p>
+              </div>
+            </div>
+          </div>
```

Comment:
This is wrong, this is general component, you need to update component at `src/app/containers/HomePage/index.js`